apps/arweave/src/ar_events_sup.erl

```diff
@@ -42,7 +42,7 @@ init([]) ->
 		?CHILD(ar_events, chunk, worker),
 		%% Events: removed
 		?CHILD(ar_events, peer, worker),
-		%% Events: initializing, account_tree_initialized, initialized,
```

Comment:
The initializing event was only handled by ar_nonce_limiter - but since so many processes listen to `node_state` it was causing a spike in memory usage during launch. On launch the block list shared via initializing can be ~300MB, so each process that listens to `node_state` (even if it doesn't handling`initializing`) receives the block list and temporarily stores it locally within the process. The GC can then take some time to clean it up - which can then cause trouble when other memory-intensive operations start up (e.g. syncing and mining)

Reply:
The fix is to remove the `initializing` event and just have `ar_node_worker` notify `ar_nonce_limiter` directly